lib/hackney/income/tenancy_gateway.rb

```
@@ -125,6 +125,25 @@ def update_tenancy(user_id:, tenancy_ref:, is_paused_until_date:, pause_reason:,
         res
       end
 
+      def get_tenancy_pause(tenancy_ref:)
+        uri = URI.parse(File.join(@api_host, "/tenancies/#{ERB::Util.url_encode(tenancy_ref)}/pause"))
+        req = Net::HTTP::Get.new(uri)
+        req['X-Api-Key'] = @api_key
+
+        res = Net::HTTP.start(uri.hostname, uri.port, use_ssl: false) { |http| http.request(req) }
+        # res = Net::HTTP.start(uri.hostname, uri.port, use_ssl: true) { |http| http.request(req) }
+
+        raise Exceptions::IncomeApiError.new(res), "when trying to get_tenancy_pause using '#{uri}'" unless res.is_a? Net::HTTPSuccess
```

we should throw a separate exception for 404 if we want to catch that one